src/common/mailFunctionality/SendMailModel.ts

```diff
@@ -106,8 +106,8 @@ type InitArgs = {
  */
 export class SendMailModel {
 	private initialized: DeferredObject<void> | null = null
-	onMailChanged: Stream<null> = stream(null)
-	onRecipientDeleted: Stream<{ field: RecipientField; recipient: Recipient } | null> = stream(null)
+	onMailChanged: Stream<null> = Stream(null)
```

Comment:
we normally import the type uppercased and the function lowercased but it's fine either way